gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,7 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-8.7-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-8.8-bin.zip
+networkTimeout=10000
```

Comment:
This should affect anything as it's only applicable to our own SDK build?

Reply:
Yep - this just related to our own SDK build